### PR TITLE
Interactive GitHub Page

### PR DESCRIPTION
**Pull request by HanzCEO**
I want people to be able to schedule their conferences using their preferable calendar app. This prototype aims to provide both HTML calendar and downloadable compiled ICS file using simple form of vanilla HTML, CSS, and JS. Hopefully, with a small start it will get even better by community contribution.

TODOs:
 - [x] Basic HTML layout
 - [x] Calendar generator
 - [x] GitHub Action to auto deploy changes
 - [x] Year selector
 - [x] Python/Node program to parse the current markdown-based schedule into JSON
 - [x] GitHub Action to automatically parse/~validate~ markdown (i'm still considering it)
 - [x] ICS file compiler (front-end)
 - [x] Rewrite to React
 - [x] Clean up mess

So far, there's no need for a back-end server. All of this can be done in client-side.

related to #143

### CODE REVIEW

File: page/src/scss/general.scss [hunk 1]
```diff
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;700&display=swap');
```
Comment (geromegrignon) [inline page/src/scss/general.scss:1]: Either add a cookie policy or add fonts directly in the project as not compliant with GDPR.

**Comment by HanzCEO** [comment]
image
<div align=center>Work In Progress Preview<div>

**Comment by HanzCEO** [comment]
I have concern regarding future collaboration on this feature.
I would like someone to rewrite this in React with cleaner code.

But anyway, this is sufficient for starting point.

**Comment by HanzCEO** [comment]
I decided to take the responsibility to clean up my code. Hence the re-draft.

**Comment by rothgar** [comment]
I just wanted to say I'm really looking forward to this PR. Especially the ics functionality. I've wanted that for a while :+1:

**Comment by HanzCEO** [comment]
image
<div align="center">Interactive year selection</div>
<br/>

image
<div align="center">Interactive date picker</div>
<br/>

image
<div align="center">Event cards with enabled badges</div>
<br/>

Alright, this should be it!

**Comment by scraly** [comment]
Wow, awesome work!

Is it possible to create a README file, a tutorial in order to use all the things you done?

Thank you ❤️

**Comment by HanzCEO** [comment]
Alright, I will make READMEs inside `page/` and `tools/` directory

**Comment by HanzCEO** [comment]
@scraly I have done adding READMEs. Hopefully it is clear enough

**Comment by HanzCEO** [comment]
After merge, this repository GitHub Page needs to be changed to the root of `gh-pages` branch. Which by a new GitHub Action will be automatically created.

**Comment by scraly** [comment]
Hi @HanzCEO , 
I merged your PR but had some troubles.
github Actions failed in error so I fixed some javascript code.

Moreover I create a `build/` folder but this one is empty.

I see a `gh-pages` branch but without any html page.

So can you check if everything is working correctly?
Thx

**Comment by scraly** [comment]
Hi @HanzCEO ,

I fixed several issues and now it's working.
The calendar is displayed, it's cool thanks :).

I have just two remarks now:
- is it possible to display all the events (of a year) without picking a date?
- is it possible to filter in a whole month?
- is it possible to filter in a whole week?
- is it possible to filter on a year when we click in a previous or next year?

Thanks

**Comment by HanzCEO** [comment]
Ah sorry for the trouble. Yes, the markdown parser is hacky and
specific so it may fail againts user-input.

I agree with point 4, but I think there shouldn't be scrolling invoked
upon year change. So the implementation simply list all events at the
bottom of the page. But I have concern with performance regarding
this, but let's do it and start small.

About monthly and weekly filter, I think someone could make a radio
selection somewhere on the page and when user select one of the option
it would change the whole page layout. From year view to monthly view
or weekly view. So just change the layout.

**Pull request merged**